spec/unit/crypto/algorithms/megolm.spec.js

```diff
@@ -68,6 +68,9 @@ describe("MegolmDecryption", function() {
             // 'crypto' implementation.
```

Comment:
This PR has fallen a bit stale and has some conflicts now. Can you merge in the latest `develop` here?

Conflicting files:

 - `spec/unit/room.spec.js`
 - `src/@types/event.ts`
 - `src/crypto/index.ts`
 - `src/models/room.ts`

Reply:
@MadLittleMods I did now a rebase :)